test/functional/JIT_Test/playlist.xml

```diff
@@ -393,4 +393,30 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+
+<!-- jit.test.recognizedMethod tests start here -->
+	<test>
+		<testCaseName>jit_recognizedMethod</testCaseName>
+		<variations>
+			<variation>-Xjit:count=1,disableAsyncCompilation,dontInline={*TestJavaLang*}</variation>
```

Comment:
I think we need an `-Xint` variation here as well to validate the test itself is correct. Is there a reason we have `dontInline` in here? These sorts of things tend to break when refactoring happens as we omit to change the names here.

Reply:
> I think we need an -Xint variation here as well to validate the test itself is correct.
Do you mean adding another test with the variation '-Xint'?

> Is there a reason we have dontInline in here?
I add the option because the function would be inlined in my previous test and I couldn't generate the log of the functions, so I add it just in case. But I think we don't need the `dontInline` here. I tried to run the command without `dontInline` option and the functions don't get inlined either. I am going to remove it.

Reply:
> Do you mean adding another test with the variation '-Xint'?

That's right.